collector-exoplayer/src/main/java/com/bitmovin/analytics/exoplayer/ExoPlayerAdapter.java

```diff
@@ -306,12 +326,43 @@ private void decorateDataWithPlaybackInformation(EventData data) {
         }
     }
 
+    @Override
+    public void onPlaybackSuppressionReasonChanged(int playbackSuppressionReason) {
+        Log.d(TAG, "onPlaybackSuppressionReasonChanged " + playbackSuppressionReason);
+    }
+
+    @Override
+    public void onIsPlayingChanged(boolean isPlaying) {
+
+    }
 
```

Comment:
Can we remove those duplicate methods?

Reply:
I checked this methods. They have same name, same number of parameters (and almost same names of parameters), but same parameters are of different type. Those methods come from event listener, and analytics listener interfaces.